extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,145 @@
+    background: var(--accent-color) !important;
+    border-radius: 32px !important;
+}
+
+.i-amp-story-quiz-prompt-container {
+    display: grid !important;
+    justify-items: center !important;
+    padding: 20px !important;
+    color: white !important;
+}
+
+.i-amp-story-quiz-prompt {
+    margin: 0px !important;
+}
+
+h1.i-amp-story-quiz-prompt {
+    font-size: 28px !important;
+    line-height: 36px !important;
+}
+
+h2.i-amp-story-quiz-prompt {
+    font-size: 22px !important;
+    line-height: 28px !important;
+}
+
+h3.i-amp-story-quiz-prompt {
+    font-size: 18px !important;
+    line-height: 24px !important;
+}
+
+.i-amp-story-quiz-option-container {
+    display: grid !important;
+    background-color: white !important;
+    border-radius: 32px !important;
+    padding: 8px !important;
+    padding-bottom: 0px !important;
+}
+
+.i-amp-story-quiz-option {
+    display: inline-grid !important;
+    grid-template-columns: 50px auto !important;
+    justify-items: start !important;
+    align-items: center !important;
+    border-radius: 32px !important;
+    padding: 8px !important;
+    padding-right: 16px !important;
+    background-color: inherit !important;
+    margin-bottom: 8px !important;
+    color: #5F6368 !important;
+    border: solid 1px !important;
+    border-color: #DADCE0 !important;
+    /* TODO: Why won't this notation work? */
+    /* font: 16px/20px !important; */
+    font-size: 16px !important;
+    line-height: 20px !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected {
+    border: none !important;
+    color: white !important;
+    padding-top: 9px !important;
+    padding-bottom: 9px !important;
+}
+
+.i-amp-story-quiz-answer-choice {
+    display: grid !important;
+    justify-items: center !important;
+    align-items: center !important;
+    height: 24px !important;
+    width: 24px !important;
+    border-radius: 50% !important;
+    border-style: solid !important;
+    border-width: 2px !important;
+    padding: 5px !important;
+    color: var(--accent-color) !important;
+    font-weight: 800 !important;
+    font-size: 16px !important;
+    line-height: 16px !important;
+}
+
+.i-amp-story-quiz-option-post-selection:not([correct]) > .i-amp-story-quiz-answer-choice {
+    color: var(--incorrect-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection[correct] > .i-amp-story-quiz-answer-choice {
+    color: var(--correct-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected:not([correct]) {
+    background-color: var(--incorrect-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected[correct] {
+    background-color: var(--correct-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected > .i-amp-story-quiz-answer-choice {
+    border: none !important;
+    padding: 7px !important;
```

Comment:
The padding being either 7, 8, or 9px is hard to follow. Can you please try other box models so you don't have to make padding adjustments?